lib/storage/src/content_manager/toc/collection_meta_ops.rs

```diff
@@ -340,7 +351,7 @@ impl TableOfContent {
             }
 
             ReshardingOperation::Abort(key) => {
-                collection.abort_resharding(key).await?;
+                collection.abort_resharding(key, false).await?;
```

Comment:
Do we want to expose the boolean to the user? Or possibly, hide it from the user, but allow us to specify it if we ever want to manually recover some state?

Reply:
Yeah, that's exactly what I wanted to do, but, I think, I'll add it later in a separate PR. Requires fiddling with consensus messages.

Reply:
Sounds good.